config/initializers/sidekiq.rb

```diff
@@ -1,34 +1,34 @@
-require 'redis'
+module SidekiqConfig
+  def self.redis_url
+    ENV.fetch('REDIS_URL', 'redis://localhost:6379/0')
+  end
+
+  def self.concurrency
+    ENV.fetch('REDIS_CONCURRENCY', 5).to_i + 2
+  end
+end
 
-config = YAML.load_file('config/redis.yml')[Rails.env]
 connection = -> {
-  Redis.new config
+  Redis.new({ url: SidekiqConfig.redis_url })
 }
 
-sidekiq_config = YAML.load_file('config/sidekiq.yml').fetch Rails.env, { }
-concurrency = sidekiq_config.fetch :concurrency, 5
-concurrency += 2 # for internal Sidekiq connections
-
-require 'sidekiq'
 Sidekiq.configure_client do |config|
-  config.redis = ConnectionPool.new size: concurrency, &connection
+  config.redis = ConnectionPool.new size: SidekiqConfig.concurrency, &connection
 end
 
 Sidekiq.configure_server do |config|
-  config.redis = ConnectionPool.new size: concurrency, &connection
+  config.redis = ConnectionPool.new size: SidekiqConfig.concurrency, &connection
```

Comment:
OK. Kind of figured they'd get picked up automatically by the gem (like AWS does) but I can't prove it, so I'll add the env var parsing to the config file. The only currently configured option is concurrency, so that's all I'll add.